### PR TITLE
Upgrade to 0.5.0

### PR DESCRIPTION
**Pull request by DunklesArchipel**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by saraedum** [comment]
@DunklesArchipel yes, you reset the build number to 0. It was 0 already, so nothing to do here.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ requirements:
     - python >=3.9
   run:
     - astropy >=5,<6
-    - click >=8,<9
+    - click ==8.0.3
```
Comment (saraedum) [inline recipe/meta.yaml:25]:
```suggestion
    - click >=8,<8.1
```

**Comment by saraedum** [comment]
Pipeline fails with:
```
Error: HTTP 404 NOT FOUND for url <[https://pypi.io/packages/source/s/svgdigitizer/svgdigitizer-0.5.0.tar.gz>](https://pypi.io/packages/source/s/svgdigitizer/svgdigitizer-0.5.0.tar.gz%3E)
```

**Comment by saraedum** [comment]
Did you upload the tarball to PyPI? @DunklesArchipel

**Comment by saraedum** [comment]
The tarballs are not uploaded automatically due to https://github.com/regro/rever/issues/254, you can upload them manually with `twine dist/svgdigitizer...0.5.0.tar.gz` or something like that.

**Comment by DunklesArchipel** [comment]
> The tarballs are not uploaded automatically due to [regro/rever#254](https://github.com/regro/rever/issues/254), you can upload them manually with `twine upload dist/svgdigitizer-0.5.0.tar.gz` or something like that.

Done! Sorry I missed that step.

**Comment by saraedum** [comment]
@DunklesArchipel once you upload the tarball, the update is picked up automatically, see #9.

**Pull request merged**